packages/create/src/index.ts

```diff
@@ -38,13 +41,17 @@ export { Libp2pComponents };
 /**
  * Create a Waku node that uses Waku Light Push, Filter and Store to send and
  * receive messages, enabling low resource consumption.
+ * Implements generics to allow for conditional type checking for Filter V1 and V2 protocols.
```

Comment:
Once nwaku with filter v2 is released and deployed, I think we can make filter v2 default method.